### ISSUE TITLE
not getting zip file

### ISSUE THREAD

**Issue by abhishek-ksolve** [miztiik/serverless-cloudwatch-logs-exporter#1]
I got this message from each log group "Permission Check Successful" in a text file inside the folder of yy/mm/..

**Comment by abhishek-ksolve** [comment]
I got  this message from each log group "Permission Check Successful"

**Comment by Y4shwanth** [comment]
yes even I implemented the same but only got "aws-logs-write-test" file with contents: Permission Check Successful , but log streams are not there in s3 buckets . there is something we need to change in python script .
